# Fix
#../block/export/fuse.c:639:19: error: 'SEEK_HOLE' undeclared (first use in this function)
#  639 |     if (whence != SEEK_HOLE && whence != SEEK_DATA) {
#      |                   ^~~~~~~~~
#../block/export/fuse.c:639:42: error: 'SEEK_DATA' undeclared (first use in this function); did you mean 'SEEK_SET'?
#  639 |     if (whence != SEEK_HOLE && whence != SEEK_DATA) {
#      |                                          ^~~~~~~~~
#      |                                          SEEK_SET
--- a/block/export/fuse.c
+++ b/block/export/fuse.c
@@ -27,7 +27,7 @@
 #include "qapi/error.h"
 #include "qapi/qapi-commands-block.h"
 #include "sysemu/block-backend.h"
-
+#include <linux/falloc.h>
 #include <fuse.h>
 #include <fuse_lowlevel.h>
